README.md

```diff
@@ -1,59 +1,80 @@
-## Instructors ##
+## Our Site: <a href="https://blue-mushroom-0022f9f1e.3.azurestaticapps.net" target="_blank">https://blue-mushroom-0022f9f1e.3.azurestaticapps.net</a>
+
+## Extra Credit
+
+### Ideas:
+
+- leader board for fewest guesses
+- add a timer to the game
```

Comment:
I think this is a neat idea that will definitely make the game harder depending on the implementation. By adding a timer it could be used for scoring purposes for the leader board idea, quickest time and fewest guesses == higher score. but another idea is using the timer to limit how long you have to think of possible words, like 30 seconds per guess, and if you go over that limit you loose a possible guess attempt.